provider/hybrid.go

```diff
@@ -56,16 +56,13 @@ func (dp dockerHybridProvider) GetSchema(ctx context.Context, request *rpc.GetSc
 }
 
 func (dp dockerHybridProvider) CheckConfig(ctx context.Context, request *rpc.CheckRequest) (*rpc.CheckResponse, error) {
-	return &rpc.CheckResponse{Inputs: request.GetNews()}, nil
+	// Delegate to the bridged provider, as native Provider does not implement it.
+	return dp.bridgedProvider.CheckConfig(ctx, request)
```

Comment:
This might have some consequences.. Since the Docker provider was written the bridge has re-enabled CheckConfig and it's really important that it gets called, as it cooperates with the Configure implementation on the bridged provider. This is a change in behavior though to enable it as it activates preconfigure callbacks if any and TF-level validators for the provider.

Reply:
It's needed for the fix since CheckConfig does the JSON secrets transcoding as well.